griptape/artifacts/image_artifact.py

```diff
@@ -37,14 +38,5 @@ def make_name(self) -> str:
 
         return f"image_artifact_{fmt_time}_{entropy}.{extension}"
 
-    @property
-    def base64(self) -> str:
-        return base64.b64encode(self.value).decode("utf-8")
-
```

Comment:
I know that we now automatically base64-encode when serializing, but I think we still need this property to be able to pull the base64-encoded image when required.

Reply:
Can we just encode at the time it is required? Like in the above example I think we can just do:
`request["init_image"] = base64.b64encode(image.value)`

Reply:
I assume the `base64` property causes issues (data duplication?) with serialization? Yeah, that's workable, just slightly less convenient.

Reply:
It just requires us implementing [this logic](https://github.com/griptape-ai/griptape/blob/dev/griptape/schemas/artifacts/image_artifact_schema.py#L17) in the automatically generated schemas. Doable, but I think the work required outweighs the minor inconvenience here.

Are there other instances of this that we need to update?